torch/_inductor/triton_heuristics.py

```diff
@@ -19,7 +19,7 @@
 import torch.autograd.profiler as autograd_profiler
 from torch._dynamo.device_interface import get_interface_for_device
 from torch._dynamo.utils import dynamo_timed
-from torch.utils._triton import has_triton, has_triton_package
+from torch.utils._triton import has_triton_package
```

Comment:
I tested it out and `get_cuda_stream` is 20x faster than `torch.cuda.current_stream`

```suggestion
from torch.utils._triton import has_triton_package
from torch._C import _cuda_getCurrentRawStream as get_cuda_stream
```

Reply:
Interesting, how did you test it?

Maybe we should bring the following logic back, in Pytorch?

```
def get_cuda_stream(idx=None):
    if idx is None:
        idx = get_current_device()
    try:
        from torch._C import _cuda_getCurrentRawStream

        return _cuda_getCurrentRawStream(idx)
    except ImportError:
        import torch

        return torch.cuda.current_stream(idx).cuda_stream
```

Reply:
The import line I suggested should be enough since we always pass `idx` and I presume the try-catch was hardening against different versions of pytorch, which obviously we have control over.

Reply:
As for how, I just used timeit.
```python
In [1]: import torch
   ...: from torch._C import _cuda_getCurrentRawStream as get_cuda_stream
   ...: device = torch.cuda.current_device()
   ...: a = torch.randn(10, device="cuda")
   ...: torch.cuda.synchronize()
   ...: %timeit get_cuda_stream(device)
   ...: %timeit torch.cuda.current_stream(device).cuda_stream
102 ns ± 0.396 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
1.83 µs ± 5.9 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```
(guess this time it's only 18x slower...)

Reply:
Thanks. BTW, how did you test the performance ? Did you run the torchbench, huggingface benchmarks?